libyul/optimiser/Suite.cpp

```diff
@@ -474,15 +474,15 @@ void OptimiserSuite::runSequence(string_view _stepAbbreviations, Block& _ast, bo
 	}
 }
 
-void OptimiserSuite::runSequence(std::vector<string> const& _steps, Block& _ast)
+void OptimiserSuite::runSequence(std::vector<std::string> const& _steps, Block& _ast)
 {
-	unique_ptr<Block> copy;
+	std::unique_ptr<Block> copy;
 	if (m_debug == Debug::PrintChanges)
-		copy = make_unique<Block>(std::get<Block>(ASTCopier{}(_ast)));
-	for (string const& step: _steps)
+		copy = std::make_unique<Block>(std::get<Block>(ASTCopier{}(_ast)));
+	for (std::string const& step: _steps)
 	{
 		if (m_debug == Debug::PrintStep)
-			cout << "Running " << step << endl;
+			std::cout << "Running " << step << "\n";
```

Comment:
```suggestion
			std::cout << "Running " << step << std::endl;
```

Same thing on lines 498, 501 and 502. With these `std::` PRs, please avoid making any other changes.

Reply:
Done